.github/workflows/ruby.yml

```diff
@@ -0,0 +1,22 @@
+name: Ruby
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        ruby: [ '2.4.10', '2.5.9', '2.6.9', '2.7.5', '3.0.3', '3.1.0' ]
```

Comment:
What versions do you actually use this with? I'd prefer not to support ruby before 2.7

Reply:
We can drop support for < 2.7.

Reply:
Done.